frontend/src/utils/webapi.js

```diff
@@ -13,6 +16,13 @@ import Node from '../Models/node.js';
 var courseId = "42";
 var mainUrl = "";
 
+// TODO(nate): This is a hardcoded user id. change this dynamically during
+// runtime based on which user is actually viewing
+var userId = 1;
+
+//TODO: this is hardcoded. fix plz
+export var classId = "if44ov1fn5a505";
```

Comment:
s/classId/piazzaCourseId/ or something like that, so we know it's specific to piazza